src/components/ComputeModal.js

```diff
@@ -686,6 +716,8 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
         hasGpu: !!gpuConfig,
         gpuType: gpuConfig?.gpuType || defaultGpuType,
         numGpus: gpuConfig?.numOfGpus || defaultNumGpus,
+        autopauseEnabled: autopauseConfig ? autopauseConfig.enabled : true,
```

Comment:
```suggestion
        autopauseEnabled: autopauseConfig?.enabled || true,
```

Reply:
I originally had it the way you're suggesting, but that caused it to default to true when enabled was false because of the or. I don't think this syntax works when both values are boolean.. Is there a better way I could do this logic?

Reply:
Oh my bad! This is actually a good use case for using the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) (`??`) instead of `||` since it uses the default value only if the predicate is nullish (i.e. `undefined` or `null`).

```suggestion
        autopauseEnabled: autopauseConfig?.enabled ?? true,
```